readthedocs/audit/templates/audit/list_logs.html

```diff
@@ -10,79 +10,105 @@
-          {% if log.log_user_id %}
-            {% blocktrans trimmed with action=log.action user=log.log_user_username path=log.resource %}
-              <a href="?user={{ user }}">
-                <code>{{ user }}</code>
-              </a>
-              <a href="?action={{ action }}" title="{{ path }}">downloaded</a> a document
+          {% elif log.action == AuditLog.INVITATION_SENT %}
+            {% blocktrans trimmed with action=log.action user=log.user_html_display|safe %}
+              User {{ user }} <a href="?action={{ action }}">sent an invitation</a>
             {% endblocktrans %}
-          {% else %}
-            {% blocktrans trimmed with action=log.action user=log.log_user_username path=log.resource %}
-              A user <a href="?action={{ action }}" title="{{ path }}">downloaded</a> a document
+          {% elif log.action == AuditLog.INVITATION_REVOKED %}
+            {% blocktrans trimmed with action=log.action user=log.user_html_display|safe %}
+              User {{ user }} <a href="?action={{ action }}">revoked an invitation</a>
+            {% endblocktrans %}
+          {% elif log.action == AuditLog.INVITATION_ACCEPTED %}
+            {% blocktrans trimmed with action=log.action user=log.user_html_display|safe %}
+              User {{ user }} <a href="?action={{ action }}">accepted an invitation</a>
+            {% endblocktrans %}
+          {% elif log.action == AuditLog.INVITATION_DECLINED %}
+            {% blocktrans trimmed with action=log.action user=log.user_html_display|safe %}
+              User {{ user }} <a href="?action={{ action }}">declined an invitation</a>
             {% endblocktrans %}
           {% endif %}
-        {% endif %}
-
-        {% trans "from" %}
-
-        <a href="?ip={{ log.ip }}" title="{{ log.browser }}">
-          <code>{{ log.ip }}</code>
-        </a>
-
-        {# If the authentication was on a doc domain. #}
-        {% if log.log_project_id %}
-          {% trans "on" %}
-          {% if log.project %}
-            <a href="{% url 'projects_detail' log.log_project_slug %}">
-              <code>{{ log.log_project_slug }}</code>
-            </a>
-          {% else %}
-            {# The original project has been deleted, don't link to it. #}
-            <code title="{{ log.resource }}">{{ log.log_project_slug }}</code>
-          {% endif %}
-        {% endif %}
+        </div>
 
         <span class="quiet right" title="{{ log.created|date:"DATETIME_FORMAT" }}">
           {% blocktrans trimmed with log.created|timesince as date %}
             {{ date }} ago
           {% endblocktrans %}
         </span>
+
+        {# Data about the log entry #}
+        <div>
+          {% if log.action == AuditLog.AUTHN or log.action == AuditLog.AUTHN_FAILURE or log.action == AuditLog.LOGOUT or log.action == AuditLog.PAGEVIEW or log.action == AuditLog.DOWNLOAD %}
+            {% if log.log_project_slug %}
+              {% trans 'project: ' %}
```

Comment:
If there isn't strong reason for it, the additional space in the translated string isn't necessary. It's not likely to be preserved in translations, and the HTML provides whitespace between the text here and the following element.

Same on occurrences below

Reply:
I thought it would be needed, but you are right, html does the spacing automatically.